instrumentation/opentelemetry-instrumentation-elasticsearch/tests/helpers_es8.py

```diff
@@ -0,0 +1,31 @@
+from elasticsearch_dsl import (  # pylint: disable=unused-import
```

Comment:
Missin license, please also add a comment that explains why importing `Keyword` is necessary here.